backend/src/common/services/slackClient.ts

```diff
@@ -19,6 +19,11 @@ export default class SlackClientService {
       new WebClient(workspace.botAccessToken)
   }
 
+  public async checkAuth(token: string) {
```

Comment:
Declare return type here instead of using `as` later on

`public async checkAuth(token: string): IAuthTestResponse  {`

Here: https://github.com/Pagepro/open-kudos/pull/80/files#diff-f07fa7010bca88e0cf5b292f910483ceR20

Reply:
Unfortounlay I can't do that because of `function checkAuth` is using slack web client library that is returning `Promise<WebAPICallResult> `. When I made changes as you mentioned I get the error:

>Type 'WebAPICallResult' is missing the following properties from type 'IAuthTestResponse': url, team, user, team_id, user_id